.github/workflows/stale.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Mark or close stale issues and PRs"
+on:
+  schedule:
+    # Run the stalebot every day at 8pm UTC
+    - cron: "00 20 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write # for actions/stale to close stale issues
+      pull-requests: write # for actions/stale to close stale PRs
+    runs-on: ubuntu-20.04
+    if: github.repository == 'alluxio/alluxio'
```

Comment:
ah this is important because the stale check action will not run on forked repos. can you add a comment stating so? we may want to make this a standard for all github action workflows

Reply:
lol forked repo will be under different names, i think this one is clear enough?